code-challenges.test.js

```diff
@@ -10,50 +10,150 @@
-const object3 = { number: -7 }
+//const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
 // b) Create the function that makes the test pass.
 
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+// --------------------2) Create a function that takes in an array of words and returns an array with 
+//                        all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// describe("capitlizer", () => {
+//   it("takes in an array of words and returns an array with all the words capitalized.", () => {
+//     expect(capitalizer(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//     expect(capitalizer(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+//   })
+// })
+
+// Psuedocode
+// Prompt: Create a function that takes in an array of words.
+
+// Input: [array]
+// Output: Capitalize all words 
+// Iterate over array using .map(), to modify and return each value/word capitalized. 
+// Note: Used other methods(only slice the first value in index, then used a method to re
+//                          rest of string to be lowerCased).
+
+// ReferenceError: capitlizer is not defined
+
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
 
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+// const capitalizer = (array) => array.map((value) => value[0].toUpperCase() + value.slice(1).toLowerCase())
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
-// a) Create a test with expect statements for each of the variables provided.
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
+// // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
-// b) Create the function that makes the test pass.
+// Psuedocode
+// Prompt: Create a function that takes in a string and logs the index of the first vowel.
+
+// Input: String
+// Output: Logs the index of the first vowel
+// Iterate over array using .flter(), to returns index of the first vowel. 
+// Note: Used other methods(only slice the first value in index, then used a method to re
```

Comment:
This looks like a repeat from the last problem's pseudocode.  It also seems that your pseudocode in regards to using filter does not match the code you provided below.  Your pseudocode should provide updates as you work through the problem.